api/src/main/java/org/pmiops/workbench/api/UserAdminController.java

```diff
@@ -139,7 +139,7 @@ public ResponseEntity<Void> createEgressBypassWindow(
   }
 
   @Override
-  @AuthorityRequired({Authority.RESEARCHER_DATA_VIEW})
```

Comment:
I don't think this was the correct authority.

RESEARCHER_DATA_VIEW is a role intended for viewing researchers' data - essentially, Workspace Admin.

SECURITY_ADMIN is what we use for other egress-related actions.